_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -42,6 +42,8 @@ For each test target, one of the following is true:
 - **semantic Role**: the test target is an [inherited][], [supported][], or [required][] [state][] or [property][] of the [semantic role][] of the element on which the test target is specified; or
 - **language feature**: the test target is specified on an [HTML element][namespaced element] and is allowed on that element. Which ARIA states or properties may be used on which element is described in [ARIA in HTML](https://w3c.github.io/html-aria/).
 
+For each test target there are no prohibited attributes for the target's role.
```

Comment:
…and I actually do not understand the need for this new expectation 🤔 

If the state or property is either global, inherited, supported, required, or explicitly allowed by ARIA in HTML, how can it be prohibited?
This needs at least a new test case to show something that fails because of this new expectation and wouldn't fail without it (see [Test every condition](https://act-rules.github.io/pages/design/test-cases/#test-every-condition) in our test cases design document).

Reply:
It does need new tests. As for why this is needed; ARIA 1.2 prohibits aria-label(ledby) on a bunch of things, even though it is a global attribute. And aria-roledescription on `generic`.

Reply:
I'm not sure I agree that it does not need new tests. Why not have a failed example with an `aria-label` on `generic` container? It passes expectation one (because it is a global property) and fails expectation 2 (because it is prohibited).

Reply:
@carlosapaduarte "It does need new tests" and that is also what @WilcoFiers wrote 😃 
(and I also had to read it twice to see there is no negation in the first sentence, I have no clue why this sentence is apparently so hard to parse 🤔)

Reply:
Damn... I read it more than twice and it always registered in my brain as "It does not need new tests" 🫣